addons/l10n_id/tests/test_qris.py

```diff
@@ -166,6 +166,8 @@ def test_fetch_payment_status_success(self):
                 self.qris_qr_invoice.message_ids[0].body,
                 Markup('<p>This invoice was paid by Zainal Arief using QRIS with the payment method Sakuku.</p>')
             )
+            # One of the QRIS transactions linked to the invoice should be paid already
+            self.assertTrue(any(self.qris_qr_invoice.l10n_id_qris_transactions.mapped('paid')))
 
     @freeze_time("2024-02-27 04:15:00")
     def test_cron_removes_outdated_transactions(self):
```

Comment:
You are no longer testing anything here, and the comments are wrong.
You can call the autovacuum method manually to test.